### DISCUSSION TITLE
An attempt to record osu lazer through OBS causes an error

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#21595] [Q&A] [answered]
Happens at any given time if you're trying to use game capture method. Windows 10 platform. Shows this error and throws a user to desktop.
![image](https://user-images.githubusercontent.com/86934170/206769385-3f0c33aa-b38d-4085-9f76-fd80d508144f.png)
[updater.log](https://github.com/ppy/osu/files/10197199/updater.log)
[network.log](https://github.com/ppy/osu/files/10197200/network.log)
[performance.log](https://github.com/ppy/osu/files/10197201/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10197202/runtime.log)

**Comment by PercyDan54** [comment]
The error code is "Access violation" which happened to me in #11599
And my solution was upgrading OBS and graphics driver

**Reply by Loreos7** [reply]
![image](https://user-images.githubusercontent.com/86934170/206850838-a7acde5f-872e-4ec9-9549-7195ad60d957.png)
![image](https://user-images.githubusercontent.com/86934170/206850845-9033c8d6-db5a-40b3-85b2-39224eabb3d4.png)

**Reply by PercyDan54** [reply]
Seems the issue happens on laptops with dual GPU, and maybe I solved it by using an external display and turning off the built-in display

**Reply by Loreos7** [reply]
> Seems the issue happens on laptops with dual GPU, and maybe I solved it by using an external display and turning off the built-in display

Already done. I always use external display with internal turned off.

**Comment by peppy** [comment, marked as answer]
Please wait for an upcoming release which will feature a new renderer, which should avoid these issues.
